airflow/providers/openlineage/extractors/base.py

```diff
@@ -144,10 +144,10 @@ def _get_openlineage_facets(self, get_facets_method, *args) -> OperatorLineage |
                 job_facets=facets.job_facets,
             )
         except ImportError:
-            self.log.exception(
+            self.log.error(
```

Comment:
```suggestion
            self.log.exception(
```
exception seems fine here